cmd/root.go

```diff
@@ -90,4 +103,6 @@ func Execute() {
 
 func init() {
 	rootCmd.Flags().Bool("gbk", false, "Use GBK decoder")
+	rootCmd.PersistentFlags().BoolP("json", "j", false, "Output in JSON format")
+	rootCmd.PersistentFlags().BoolP("jsonline", "l", false, "JSON output with newlines")
```

Comment:
`PersistentFlags` should not be used here. These flags are unreasonable under some subcommands.

```
❯ ./nali info --help
get the necessary information of nali

Usage:
  nali info [flags]

Examples:
nali info

Flags:
  -h, --help   help for info

Global Flags:
  -j, --json       Output in JSON format
  -l, --jsonline   JSON output with newlines
```

Reply:
sure, I will change it.